sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/parser/ParseDriver.scala

```diff
@@ -88,19 +89,24 @@ abstract class AbstractSqlParser(conf: SQLConf) extends ParserInterface with Log
   protected def parse[T](command: String)(toResult: SqlBaseParser => T): T = {
     logDebug(s"Parsing command: $command")
 
+    val useSQLStandardKeywords = Dialect.withName(conf.dialect) match {
```

Comment:
maybe `conf.dialect` should return `Dialect`.

Reply:
Sure, done in f16b6ea.